### PR TITLE
eh - DriverAvailabilityForm + tests + stories

### LINKED ISSUE #12: EPIC: Driver Availability - #3 DriverAvailabilityForm
DriverAvailabilityForm

- [ ] There is a react component DriverAvailabilityForm that is suitable for use on the pages described in the acceptance criteria

 - [ ] There are storybook entries

### PR DESCRIPTION
**Pull request by elaineeeho**
Implemented the DriverAvailabilityForm, tests for the form, and stories implementation
Screenshot 2024-02-29 at 8 02 32 PM

Closes #12

### CODE REVIEW

**Review by andy-gale — DISMISSED** [review]
Ran locally. Looks good to me.

**Review by ashton-wong — DISMISSED** [review]
LGTM

File: frontend/src/main/components/Driver/DriverAvailabilityForm.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    );
+    // Stryker restore all
+    const navigate = useNavigate();
+
+    const testIdPrefix = "DriverAvailabilityForm";
+    return (
+        <Form onSubmit={handleSubmit(submitAction)}>
+            {initialContents && (
+                <Form.Group className="mb-3" >
+                    <Form.Label htmlFor="id">id</Form.Label>
```
Comment (andrewpengucsb) [inline frontend/src/main/components/Driver/DriverAvailabilityForm.js:22]: Form labels should be capitalized, lowercase is a bit unprofessional and is inconsistent with other forms in the application

**Review by andrewpengucsb — CHANGES_REQUESTED** [review]

**Review by michaelw1985 — APPROVED** [review]
LGTM

**Review by andrewpengucsb — APPROVED** [review]

**Pull request merged**